qemu/tests/cfg/kvm_unit_test_nested.cfg

```diff
@@ -0,0 +1,13 @@
+- kvm_unit_test_nested:
+    type = kvm_unit_test_nested
+    start_vm = no
+    vms = ""
```

Comment:
BTW, I see the manual case requested to boot L1 and L2 guest in the host, but this patch could not cover them. Is it expected?

Reply:
The test suite will start a qemu process for the tests itself, and will handle l2 guests,  no need to start other guests.

Reply:
If so, you may need to update the manual test steps.
@CongLi, I am not sure the usage from the x86 side, you guys can confirm this item together.

Reply:
@PaulYuuu Yes, no need to boot guest, it's only check nested function in host.
@qcheng-redhat Please keep the automation case is same as the polarion case steps.

Reply:
I have updated test case in Polarion. @PaulYuuu @CongLi 
https://polarion.engineering.redhat.com/polarion/#/project/RedHatEnterpriseLinux7/workitem?id=RHEL7-19680

Reply:
Updated with starting 4 l1 guest vms on host.